remotemonitoring/scripts/all-in-one.yaml

```diff
@@ -194,7 +194,7 @@ spec:
     spec:
       containers:
       - name: device-simulation-pod
-        image: azureiotpcs/device-simulation-dotnet:testing
+        image: azureiotpcs/device-simulation-dotnet:{release}
```

Comment:
Docker image tags are going to diverge, e.g. we'll have 
* 1.0.0-preview.2
* 1.0.3-preview.2
as soon as someone will send a PR.
Can we use the release number to identify a docker-compose and k8s configuration instead?

Reply:
e.g. something like
* docker-compose.testing.dotnet.yaml
* docker-compose.1.0.0-preview.1.dotnet.yaml
* docker-compose.1.0.0-preview.2.dotnet.yaml
etc.

Reply:
I would suggest for now based on our discussion earlier that we rollover version numbers for all the images as long as we are in preview. We could use the release number on 'RemoteMonitoring' solution to figure out what individual docker image versions for the microservices are. Not sure if that is what you meant by using release number.